src/main/scala/wdl4s/cwl/CommandLineTool.scala

```diff
@@ -44,18 +42,18 @@ case class CommandInputEnumSchema(
 
 case class CommandInputArraySchema(
                                     items:
-                                      CwlType :+:
+                                    CwlType :+:
                                       CommandInputRecordSchema :+:
                                       CommandInputEnumSchema :+:
                                       CommandInputArraySchema :+:
                                       String :+:
                                       Array[
                                         CwlType :+:
-                                        CommandInputRecordSchema :+:
-                                        CommandInputEnumSchema :+:
-                                        CommandInputArraySchema :+:
-                                        String :+:
-                                        CNil] :+:
+                                          CommandInputRecordSchema :+:
+                                          CommandInputEnumSchema :+:
+                                          CommandInputArraySchema :+:
+                                          String :+:
+                                          CNil] :+:
```

Comment:
Can you please revert this as all of these types are potential choices of the array, shouldn't be separate from `CwlType` above